### ISSUE TITLE
Latest github action runners `2.306.0` cause workflow to fail

### ISSUE THREAD

**Issue by gidjin** [google-github-actions/release-please-action#793]
### TL;DR

Our github runners upgraded to version `2.306.0` from version `2.305.0` and we started having the following error with our release please action

```sh
✔ Considering: 133 commits
❯ component: 
❯ pull request title pattern: undefined
Error: release-please failed: types.find is not a function
```

Failed Build example: https://github.com/trussworks/react-uswds/actions/runs/5548450381/jobs/10131440378
Working Build: https://github.com/USSF-ORBIT/ussf-portal-client/actions/runs/5512846343/jobs/10050194817
Failed Build: https://github.com/USSF-ORBIT/ussf-portal-client/actions/runs/5535147685/jobs/10101056417

### Expected behavior

Release please action to run and create a release branch

### Observed behavior

Workflow fails because `types.find is not a function`

### Action YAML

```yaml
on:
  push:
    branches:
      - main
  workflow_dispatch:


permissions:
  contents: write
  pull-requests: write

name: release-please

jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          release-type: node
          package-name: '@trussworks/react-uswds'
          release-labels: 'type: release'
          changelog-types: >
            { "types":
              [
                { "type": "build", "hidden": true },
                { "type": "chore", "hidden": true },
                { "type": "ci", "hidden": true },
                { "type": "feat", "section": "Features" },
                { "type": "fix", "section": "Bug Fixes" },
                { "type": "docs", "section": "Documentation & Examples" },
                { "type": "style", "hidden": true },
                { "type": "refactor", "hidden": true },
                { "type": "perf", "section": "Performance Improvements" },
                { "type": "test", "hidden": true }
              ]
            }
```

### Log output

```text
Run google-github-actions/release-please-action@51ee8ae2605bd5ce1cfdcc5938684908f1cd9f69
✔ Looking for latest release on branch: main with prefix: ussf-portal-client
❯ Fetching merge commits on branch main with cursor: undefined
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 24
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 49
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 74
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 99
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 124
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 149
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 174
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 199
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 224
❯ Fetching releases with cursor undefined
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 22,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 21,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 20,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 19,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 18,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 18,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 17,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 16,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 15,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 14,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 13,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 12,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 12,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 11,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 10,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 9,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 9,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 8,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 7,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 6,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 5,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 4,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 3,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 2,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 1,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ Fetching releases with cursor Y3Vyc29yOnYyOpK0MjAyMi0wNS0xOFQyMDo0MjoyM1rOBAGxRg==
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 0,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 0,
  patch: 0,
  preRelease: 'beta.0',
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 3,
  minor: 1,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 3,
  minor: 0,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 5,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 4,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 3,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 2,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 1,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 0,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 0,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 0,
  patch: 0,
  preRelease: 'alpha.1',
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 0,
  patch: 0,
  preRelease: 'alpha.0',
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 1,
  minor: 3,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 1,
  minor: 0,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 1,
  minor: 0,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found 18 possible releases. [
  Version {
    major: 4,
    minor: 22,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 21,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 20,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 19,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 18,
    patch: 1,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 18,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 17,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 16,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 15,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 14,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 13,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 12,
    patch: 1,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 12,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 11,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 10,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 9,
    patch: 1,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 9,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 8,
    patch: 0,
    preRelease: undefined,
    build: undefined
  }
]
✔ Building releases
✔ Building strategies by path
❯ .: node
✔ Looking for latest release on branch: main with prefix: ussf-portal-client
❯ Fetching merge commits on branch main with cursor: undefined
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 24
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 49
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 74
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 99
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 124
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 149
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 174
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 199
❯ Fetching merge commits on branch main with cursor: 4e49251ae7261e45a981c59e8653ba8c8bc57f2e 224
❯ Fetching releases with cursor undefined
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 22,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 21,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 20,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 19,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 18,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 18,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 17,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 16,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 15,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 14,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 13,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 12,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 12,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 11,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 10,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 9,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 9,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 8,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 7,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 6,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 5,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 4,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 3,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 2,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 1,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ Fetching releases with cursor Y3Vyc29yOnYyOpK0MjAyMi0wNS0xOFQyMDo0MjoyM1rOBAGxRg==
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 0,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 4,
  minor: 0,
  patch: 0,
  preRelease: 'beta.0',
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 3,
  minor: 1,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 3,
  minor: 0,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 5,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 4,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 3,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 2,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 1,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 0,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 0,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 0,
  patch: 0,
  preRelease: 'alpha.1',
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 2,
  minor: 0,
  patch: 0,
  preRelease: 'alpha.0',
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 1,
  minor: 3,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 1,
  minor: 0,
  patch: 1,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found release for ussf-portal-client Version {
  major: 1,
  minor: 0,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ SHA not found in recent commits to branch main, skipping
❯ found 18 possible releases. [
  Version {
    major: 4,
    minor: 22,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 21,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 20,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 19,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 18,
    patch: 1,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 18,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 17,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 16,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 15,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 14,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 13,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 12,
    patch: 1,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 12,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 11,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 10,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 9,
    patch: 1,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 9,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 4,
    minor: 8,
    patch: 0,
    preRelease: undefined,
    build: undefined
  }
]
✔ Building pull requests
✔ Building strategies by path
❯ .: node
✔ Collecting release commit SHAs
❯ release search depth: 400
❯ Fetching releases with cursor undefined
❯ Found release for path ., v4.22.0
❯ release for path: ., version: 4.22.0, sha: 9b382006c77a4d19cdbc95a93a6[35](https://github.com/USSF-ORBIT/ussf-portal-client/actions/runs/5535147685/jobs/10101056417#step:2:36)aaddd4098fe
✔ Collecting commits since all latest releases
❯ commit search depth: 500
❯ Set(1) { '9b[38](https://github.com/USSF-ORBIT/ussf-portal-client/actions/runs/5535147685/jobs/10101056417#step:2:39)2006c77a4d19cdbc95a93a635aaddd[40](https://github.com/USSF-ORBIT/ussf-portal-client/actions/runs/5535147685/jobs/10101056417#step:2:41)98fe' }
❯ Fetching merge commits on branch main with cursor: undefined
✔ Splitting 1 commits by path
✔ Building candidate release pull request for path: .
❯ type: node
❯ targetBranch: main
❯ commits: 1
✔ Considering: 1 commits
❯ component: 
❯ pull request title pattern: undefined
Error: release-please failed: types.find is not a function
```

**Comment by gidjin** [comment]
After some debugging and walking co-workers through the issue we found that my configuration was incorrect for changelog-types. Updating it as follows has resolved the issue. No bug but leaving this note here in case someone else has the same issue.

```yaml
on:
  push:
    branches:
      - main
  workflow_dispatch:


permissions:
  contents: write
  pull-requests: write

name: release-please

jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          release-type: node
          package-name: '@trussworks/react-uswds'
          release-labels: 'type: release'
          changelog-types: >
            [
              { "type": "build", "hidden": true },
              { "type": "chore", "hidden": true },
              { "type": "ci", "hidden": true },
              { "type": "feat", "section": "Features" },
              { "type": "fix", "section": "Bug Fixes" },
              { "type": "docs", "section": "Documentation & Examples" },
              { "type": "style", "hidden": true },
              { "type": "refactor", "hidden": true },
              { "type": "perf", "section": "Performance Improvements" },
              { "type": "test", "hidden": true }
            ]
```
